packages/odyssey-react-mui/src/Banner.tsx

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { AlertColor } from "@mui/material";
+import { Alert, Link } from "./";
+
+export interface BannerProps {
+  /**
+   * Determine the color and icon of the alert
+   */
+  severity?: AlertColor;
+  /**
+   * Sets the ARIA role of the alert
+   * ("status" for something that dynamically updates, null for something
+   * unchanging)
+   */
+  role?: "status" | undefined;
```

Comment:
I figured out what I want here. We should use `AlertProps["role"]` since its' a pass-through props.

If it's a prop we don't control that we simply pass-through, it makes more sense for the consuming component to define the type, not our component.

There's a problem with this approach:
If we start adding functionality in our parent component for this prop, the type is now tied to a child component rather than our own unless someone knows to change it.

But in general, it's a good pattern to avoid tight coupling. If MUI changes their prop, and we have to change ours, that's tight coupling. Ideally, we would set our type to `unknown` or `any` and let MUI handle it, but TypeScript doesn't work that way like runtime PropTypes did.

So in this case, it's best if we let MUI handle the types of any pass-through props like `AlertProps["role"]` rather than re-define those ourselves.

Reply:
MUI's type here is `string`, but we don't want people passing in *any* role for Banner, only `status` if necessary. Should we still hinge on MUI's props in that case?

Reply:
I guess, why are we passing status or not? Why isn't that hardcoded for `Banner`?

Reply:
This is from a conversation Shred and I had last week:
<img width="697" alt="image" src="https://user-images.githubusercontent.com/91091570/213807108-162e4bc3-1cc8-44ba-873a-1c29930b5dfd.png">